solidity/contracts/bridge/BridgeState.sol

```diff
@@ -91,6 +91,13 @@ library BridgeState {
         // than `redemptionDustThreshold` in order to prevent redemption requests
         // with values lower or equal to `movingFundsDustThreshold`.
         uint64 movingFundsDustThreshold;
+        // Time after which the moving funds timeout can be reset in case the
+        // target wallet commitment cannot be submitted due to a lack of live
+        // wallets in the system. It is counted from the moment when the wallet
+        // was requested to move their funds and switched to the MovingFunds
+        // state. Value in seconds. This value should be smaller than the value
```

Comment:
```suggestion
        // state. Value in seconds. This value should be lower than the value
```

See [this](https://github.com/keep-network/tbtc-v2/pull/241#discussion_r870114337) conversation please.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/282/commits/90f362a87287334b6c65ebdcc7d04342827155e1